tools/torch/main.lua

```diff
@@ -181,8 +181,8 @@ if opt.subtractMean ~= 'none' then
 end
 
 local classes
-local confusion
-local validation_confusion
+local trainConfusion
+local validConfusion
```

Comment:
Yes I think `val` is good. It's short and it's already in other parts of the code to denote the validation phase. Thanks!